chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -35,12 +35,8 @@ props = c['attributes']
 </style>
 <!-- html output -->
     <tr>
-      <th class="cell-left">${_('tt_bfe_hoehe_ueber_grund')}</th>
-      <td>${altitude or '-'} m</td>
-    </tr>
-    <tr>
-      <th class="cell-left">${_('tt_bfe_koordinaten')}</th>
-      <td>${center}</td>
+      <th class="cell-left">${_('tt_bfe_hinweis_wa')}</th>
+      <td>${_('tt_bfe_hinweis_text')}</td>
```

Comment:
Is that wanted that you've removed "Altitude above ground" and "Coordinates (X/Y)" ? If so we could remove the translations from the already big translation file.

Reply:
Actually no, do not remove them. Those strings are still needed in the extended tooltip (same mako)